include/tools.h

```diff
@@ -38,22 +38,28 @@ typedef std::vector<std::string> FeedCategories;
 std::string getCurrentDirectory();
 
 /**
- * Return the data directory.
- *
- * The data directory is a directory where to put data (zim files, ...)
- * It depends of the platform and it may be changed by user using environment variable.
- *
- * The resolution order is :
- * - `KIWIX_DATA_DIR` env variable (if set).
- * - On Windows :
- *    . `$APPDATA/kiwix` if $APPDATA is set
- *    . `$USERPROFILE/kiwix` if $USERPROFILE is set
- * - Else :
- *    . `$XDG_DATA_HOME/kiwix`if $XDG_DATA_HOME is set
- *    . `$HOME/.local/share/kiwx` if $HOWE is set
- * - current directory
- *
- * @return the path of the data directory (utf8 encoded)
+ * Return the data directory
+ *
+ * The data directory is foreseen to put ZIM files (and temporary
+ * files during download).  Its path can vary and depends on a few
+ * things.
+ *
+ * Resolution order is:
```

Comment:
The following is a more accurate description of how the data directory is used:

> The data directory is the default directory where downloaded files should be saved (it can be overriden via the `options` parameter of `kiwix::Downloader::startDownload()`).
> 
> Its path can vary and is determined as follows:

Reply:
Will fix and merge.